.github/workflows/ci.yml

```diff
@@ -158,7 +158,7 @@ jobs:
             Thank you for your contribution.
             You can use the following dependency to test your application while we review your pull request:
 
-            `com.github.Skyscanner:backpack-android:${{ github.sha }}`
+            `net.skyscanner.backpack:backpack-android:${{ github.sha }}`
```

Comment:
how does this actually work? i don't see anything actually doing the publishing?!